lib/src/runner/iojs_version.dart

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2015, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// io.js version parser.
+library dev_compiler.src.runner.iojs_version;
+
+/// Comparable representation of an io.js version (see https://iojs.org/).
+///
+/// Note that io.js is the future of node.js:
+/// http://www.infoq.com/news/2015/05/nodejs-iojs
+class IoJsVersion extends Comparable {
```

Comment:
FYI, pub_semver package has a Version object. We use it in our SDK version check script:
https://github.com/dart-lang/dev_compiler/blob/902cab3c54226b28548140ada0c59d4da48cb8c2/tool/sdk_version_check.dart#L21